extensions/blocks/social-previews/constants.js

```diff
@@ -1,24 +1,42 @@
 /**
  * External dependencies
  */
+import { Fragment } from 'react';
```

Comment:
please import this from `@wordpress/element` instead of `react` directly

Reply:
Great suggestion 
https://github.com/Automattic/jetpack/pull/16745/commits/ca902c3ea7861834678ebec9b90674f17026b6af